includes/admin/views/onboarding-page-one.php

```diff
@@ -19,17 +19,24 @@
 			<div class="simple-podcasting__setting-description"><?php esc_html_e( 'What’s the title of your podcast show that listeners will see?', 'simple-podcasting' ); ?></div>
 		</div>
 
+		<!-- Show artist/author -->
+		<div class="simple-podcasting__setting">
+			<label class="simple-podcasting__setting-label"><?php esc_html_e( 'Show Artist / Author name*', 'simple-podcasting' ); ?></label>
+			<div class="simple-podcasting__setting-input"><input name="podcast-artist" required type="text" /></div>
+			<div class="simple-podcasting__setting-description"><?php esc_html_e( 'Who’s the artist or author of your podcast show that listeners will see?', 'simple-podcasting' ); ?></div>
+		</div>
+
 		<!-- Show description -->
 		<div class="simple-podcasting__setting">
-			<label class="simple-podcasting__setting-label"><?php esc_html_e( 'Show description', 'simple-podcasting' ); ?></label>
-			<div class="simple-podcasting__setting-input"><textarea name="podcast-description" rows="5"></textarea></div>
+			<label class="simple-podcasting__setting-label"><?php esc_html_e( 'Show description*', 'simple-podcasting' ); ?></label>
+			<div class="simple-podcasting__setting-input"><textarea name="podcast-description" rows="5" required></textarea></div>
```

Comment:
```suggestion
			<label class="simple-podcasting__setting-label" for="simple-podcasting-podcast-description"><?php esc_html_e( 'Show description*', 'simple-podcasting' ); ?></label>
			<div class="simple-podcasting__setting-input"><textarea name="podcast-description" id="simple-podcasting-podcast-description" rows="5" required></textarea></div>
```

Accessibility

You'll also need [aria-describedby ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)for the descriptive text.